src/main/java/apoc/coll/Coll.java

```diff
@@ -431,6 +432,18 @@ public boolean containsAllSorted(@Name("coll") List<Object> coll, @Name("values"
         return true;
     }
 
+    @UserFunction
+    @Description("apoc.coll.isEqualCollection(coll, values) return true if two collections contain the same elements with the same cardinality in any order (using a HashMap)")
+    public boolean isEqualCollection(@Name("coll") List<Object> first, @Name("values") List<Object> second) {
+        if (first == second) return true;
```

Comment:
@vboulaye please remove this line or use `equals` after the null check

Reply:
the problem is that if I remove this test, I have to add an explicit test to handle the case when both entries are null:  I believe  ̀apoc.coll.isEqualCollection(null,null)`  should return true .
I added 
```
        if (first == null && second == null) return true;
```
but I am not sure it is much better!

Reply:
I just saw that in containsAllSorted the null check is done  with   if (coll == null || values == null) return false;
so containsAllSorted(null,null) is false

maybe this should be the same for  isEqualCollection ?